src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -311,6 +311,129 @@ def k8s_install_cli(client_version='latest', install_location=None):
+                                              dns_name_prefix, location, connector_name)
+    client_secret = principal_obj.get("client_secret")
+    service_principal = principal_obj.get("service_principal")
+    # Get the TenantID
+    profile = Profile()
+    _, _, tenant_id = profile.get_login_credentials()
+    # Check if we want the windows connector
+    if (os_type == 'windows') or (os_type == 'both'):
+        # The current connector will deploy two connectors, one for Windows and another one for Linux
+        image_tag = 'canary'
+    logger.warning('Deploying the aci-connector using Helm')
+    try:
+        subprocess.call(["helm", "install", url_chart, "--name", connector_name, "--set", "env.azureClientId=" +
+                         service_principal + ",env.azureClientKey=" + client_secret + ",env.azureSubscriptionId=" +
+                         subscription_id + ",env.azureTenantId=" + tenant_id + ",env.aciResourceGroup=" + rgkaci.name +
+                         ",env.aciRegion=" + location + ",image.tag=" + image_tag])
+    except subprocess.CalledProcessError as err:
+        raise CLIError('Could not deploy the ACI Chart: {}'.format(err))
+
+
+def k8s_uninstall_connector(client, name, connector_name, resource_group,
+                            graceful=False, os_type='Linux'):
+    """Undeploy the ACI-Connector from an AKS cluster.
+    :param name: The name of the AKS cluster. The name is case insensitive.
+    :type name: str
+    :param resource_group: The name of the resource group where the AKS cluster is deployed.
+    :type resource_group: str
+    :param connector_name: The name for the ACI Connector
+    :type connector_name: str
+    """
+    from subprocess import PIPE, Popen
+    helm_not_installed = "Error : Helm not detected, please verify if it is installed."
+    # Check if Helm is installed locally
+    try:
+        Popen(["helm"], stdout=PIPE, stderr=PIPE)
+    except OSError:
+        raise CLIError(helm_not_installed)
+    # Get the credentials from a AKS instance
+    _, browse_path = tempfile.mkstemp()
+    aks_get_credentials(client, resource_group, name, admin=False, path=browse_path)
+    # Validate if the RG exists
+    # Just do the get, we don't need the result, it will error out if the group doesn't exist.
+    if graceful:
+        logger.warning('Graceful option selected, will try to drain the node first')
+        kubectl_not_installed = "Kubectl not detected, please verify if it is installed."
+        try:
+            Popen(["kubectl"], stdout=PIPE, stderr=PIPE)
+        except OSError:
+            raise CLIError(kubectl_not_installed)
+        try:
+            if (os_type == 'windows') or (os_type == 'both'):
+                drain_node = subprocess.check_output(
+                    ["kubectl", "drain", "aci-connector-0", "--force"],
+                    universal_newlines=True)
+                drain_node = subprocess.check_output(
```

Comment:
suggest use a loop to avoid code dupes:
```python
 nodes = ['-0', '-1'] if os_type.lower in ['windows', 'both'] else ['']
 for i in nodes:
    drain_node = subprocess.check_output(["kubectl", "drain", "aci-connector{}".format(i), "--force"],  universal_newlines=True)
```